.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+<!--- PR title format: [GH#<gh-issue-id>][BZ#<bz-issue-id>][<jira-issue-id>]: <short-description-of-the-pr> --->
+
+Version:
+<!--- What version or versions of OpenShift does this apply to? --->
+
+Issue:
+<!--- Add a link to the Bugzilla, Jira, or GitHub issue. --->
+
+Link to docs preview:
+<!--- Add direct links to the Netlify preview build. --->
+
+Description:
+
+<!--- For more information about contributing to OpenShift documentation, see:
```

Comment:
I'd suggest to start this whole section off with the most important thing that the submitter needs to know (IMO that is to submit this PR for review).

Maybe we could consider adding a (hidden) heading ("After submitting your PR" or something) with the two methods of pinging the squad for review? And then after that list the highlights of what we will expect (squashed commits, QE review, links to the contrib resources)